data.js

```diff
@@ -669,6 +669,14 @@ module.exports = [
     url: "https://github.com/PoeHaH/devicedetector",
     source: "https://raw.githubusercontent.com/PoeHaH/devicedetector/master/devicedetector-production.js"
   },
+  {
+    name: "svelte-device-info",
+    github: "rozek/svelte-device-info",
+    tags: ["mobile", "phone","tablet","form-factor","resolution-touch"],
+    description: "informs about a device's form factor and pointing accuracy (not only in Svelte)",
+    url: "https://github.com/rozek/svelte-device-info",
+    source: "https://raw.githubusercontent.com/rozek/svelte-device-info/main/src/svelte-device-info.ts"
```

Comment:
From the readme - emphasis on "ready-to-use", which means you should not link to a TypeScript source file:

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified (see below)